.github/workflows/tests.yaml

```diff
@@ -0,0 +1,41 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master, v5 ]
+  pull_request:
+    branches: [ master, v5 ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  ansible:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install Ansible
+        run: |
+          python -m pip install --upgrade wheel
+          python -m pip install --upgrade ansible
+
+      - name: Run the all-in-one playbook
+        run: |
+          if test -f ansible/all-in-one/site.yml; then
+            # v5
```

Comment:
nit: I think we should just create `version/4` branch from master, and assume we're dealing with `v5` only.

Reply:
Yeah, I've started wondering that too as soon as I sent this PR. Let me update it to cover `v4` and then create that branch (the first commit that sets up GH Workflows must go to the default branch)